site/zenodo_rdm/subcommunities/request.py

```diff
@@ -6,21 +6,35 @@
 # it under the terms of the MIT License; see LICENSE file for more details.
 """Subcommunities request implementation for ZenodoRDM."""
 
+import invenio_communities.notifications.builders as notifications
 from invenio_access.permissions import system_identity
+from invenio_communities.proxies import current_communities
 from invenio_communities.subcommunities.services.request import (
-    AcceptSubcommunity,
-    DeclineSubcommunity,
-    SubCommunityRequest,
-)
-from invenio_rdm_records.proxies import (
-    current_community_records_service,
-    current_rdm_records,
-)
+    AcceptSubcommunity, AcceptSubcommunityInvitation,
+    CreateSubcommunityInvitation, DeclineSubcommunity,
+    DeclineSubcommunityInvitation, SubCommunityInvitationRequest,
+    SubCommunityRequest)
+from invenio_notifications.services.uow import NotificationOp
+from invenio_rdm_records.proxies import (current_community_records_service,
+                                         current_rdm_records)
 from invenio_records_resources.services.uow import RecordCommitOp
 from invenio_requests.customizations import actions
 from invenio_requests.customizations.event_types import CommentEventType
 from invenio_requests.proxies import current_events_service
 from marshmallow import fields
+from werkzeug.local import LocalProxy
+
+community_service = LocalProxy(lambda: current_communities.service)
```

Comment:
just out of curiosity, why additional localProxy wrap is needed?

Reply:
community_service isn't used so I think this is a mistake, thanks!